packages/dashboard/public/index.html

```diff
@@ -4,6 +4,16 @@
         <meta charset="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <title>Mask Network</title>
+        <link rel="dns-prefetch" href="https://files.r2d2.to" />
```

Comment:
No need to prefetch in the dashboard. Because all request will be sent by the background page and it will do the dns-prefetch

Reply:
where?

Reply:
This page is used by the new dashboard. To make the new dashboard works, the user must have Mask Network installed. And you have already contained those DNS prefetches in the background page so those URLs are already prefetched.